user/trusty-ci-environment.md

```diff
@@ -0,0 +1,278 @@
+
+#### Node.JS
+
+[nvm](https://github.com/creationix/nvm#installation) is installed and we've pre-installed the following Node versions
+with it
+
+- iojs-v1.6.4
+- v0.6.21
+- v0.8.28
+- v0.10.40
+- v0.11.16
+- v0.12.7
+- 4.1.2
+
+#### Go
+
+[gimme](https://github.com/travis-ci/gimme#gimme) is installed and we've pre-installed the following Go versions
+with it
+
+- go 1.0.3
+- go 1.1.2
+- go 1.2.2
+- go 1.3.3
+- go 1.4.2
+- go 1.5.1
+
+#### JVM
+
+- We install the latest versions OpenJDK versions from the official Ubuntu
+  Trusty packages. (1.8.0_60 as of 2015-10-13)
+- We install the latest Oracle JDK versions from Oracle.
+- [jdk_switcher](https://github.com/michaelklishin/jdk_switcher#what-jdk-switcher-is) is installed if you need another version.
+- gradle 2.5 is installed.
+- Maven 3.1.1 is installed.
+- leiningen 2.5.1 is installed.
+- sbt 0.13.9
+
+#### PHP
+
+We do not currently have any PHP versions pre-installed in the **mega**
+image. This is at the top of our list for our next update.
+
+
+But we do have [phpenv](https://github.com/phpenv/phpenv) installed and
+it's able to install the following PHP versions.
+
+- PHP 5.3
+- PHP 5.4
+- PHP 5.5
+- PHP 5.6
+- PHP 7.0.0 RCs
+
+_Note: We're unable to build **PHP 5.2** on Trusty so far, so trying to use
+it will result in a build failure when phpenv fails to compile it_
+
+#### Other runtimes
+
+Other runtimes are available to install using `apt-get` and the official
+Ubuntu packages, any third party PPAs or custom scripts that you
+manually setup in your `.travis.yml`
+
+### Data Stores
+
+We pre-install the following services and they can be activated with the built-in [services](/user/database-setup/) support.
+
+* PostgreSQL 9.4.5
+* SQLite 3.8.2
+* Redis 3.0.4
+
+*Note: The following services that are included with our Precise images
+are **not included** in the Trusty **mega** images at this time but will
+be added in the coming months. You'll need to install them manually for
+the time being.*
+
+- MySQL
+- Riak
+- ElasticSearch
+- Apache Cassandra
+- CouchDB
+- Neo4J
+
+### Firefox
+
+Firefox 31.6.0esr is installed by default.
+
+If you need a specific version of Firefox, use the Firefox addon to install
+it during the `before_install` stage of the build.
+
+For example, to install version 17.0, add the following to your
+`.travis.yml` file:
+
+    addons:
+      firefox: "17.0"
+
+### Headless Browser Testing Tools
+
+* [xvfb](http://en.wikipedia.org/wiki/Xvfb) (X Virtual Framebuffer)
+* [PhantomJS](http://phantomjs.org/)
+
+### Environment variables
+
+There is a [list of default environment variables](/user/environment-variables#Default-Environment-Variables) available in each build environment.
+
+### apt configuration
+
+apt is configured to not require confirmation (assume -y switch by default) using both `DEBIAN_FRONTEND` env variable and apt configuration file. This means `apt-get install -qq` can be used without the -y flag.
+
+### Group membership
```

Comment:
This section feels out of place to me, but still super useful.  Could it be moved elsewhere given that it applies to precise and trusty alike?